src/libraries/System.IO.FileSystem.AccessControl/src/System/IO/FileSystemAclExtensions.cs

```diff
@@ -50,23 +50,43 @@ public static void SetAccessControl(this FileInfo fileInfo, FileSecurity fileSec
             fileSecurity.Persist(fullPath);
         }
 
+        /// <summary>
+        /// This extension method for FileStream returns a FileSecurity object containing security descriptors from the Access, Owner, and Group AccessControlSections.
+        /// </summary>
+        /// <param name="fileStream">An object that represents the file for retrieving security descriptors from</param>
+        /// <exception cref="ArgumentNullException">is <see langword="null" />.</exception>
+        /// <exception cref="ObjectDisposedException">File is closed.</exception>
```

Comment:
The file stream is what's closed:
```suggestion
        /// <exception cref="ObjectDisposedException">The file stream is closed.</exception>
```